auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -661,7 +661,7 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
 // Custom Scroll bars
 #affiliated-entity-table {
   ::v-deep .v-menu__content {
-    margin-left: -6rem;
+    margin-left: 1rem;
```

Comment:
Alex and I played with this and found that this CSS is not the correct way to align the v-menu with the right side of the button, and this limits the fix that can be done here to prevent the button misalignment we saw earlier.

We decided to take a compromise approach that avoids the misalignment issue and doesn't force us to fix the CSS right now. But it's not perfect... We created the following ticket to implement a better/correct solution for aligning the v-menus:

https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/12162

Reply:
Alex and I played with this and found that this CSS is not the correct way to align the v-menu with the right side of the button, and this limits the fix that can be done here to prevent the button misalignment we saw earlier.

We decided to take a compromise approach that avoids the misalignment issue and doesn't force us to fix the CSS right now. But it's not perfect... We created the following ticket to implement a better/correct solution for aligning the v-menus:

https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/12162